setup-tpm2-totp.sh

```diff
@@ -0,0 +1,30 @@
+
+sudo apt -y install build-essential autoconf autoconf-archive automake m4 libtool gcc pkg-config libqrencode-dev pandoc doxygen liboath-dev iproute2 plymouth libplymouth-dev libssl-dev libjson-c-dev libcurl4-openssl-dev
```

Comment:
Fwiw pandoc, doxygen, liboath-dev, and I think iproute2 are dev-only dependencies, which I'm pretty sure we don't need.

Reply:
but we need them for the build right? So you're saying we can remove them afterwards?

Reply:
I don't think we do need them for build. The configure step will turn off the docs building parts of the build